24H Study Room/Models/UserCredential.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UserCredential.swift
+//  24H Study Room
+//
+//  Created by Jia-Jiuan Tsai on 2022-09-08.
+//
+
+import Foundation
+import FirebaseAuth
+
+struct UserCredential {
```

Comment:
[Naming] This name is misleading. The term "user credentials" refers to data like email, username, password and similar. This struct does not contain any _user credentials_, but the actual model of the _user_. 

Suggested names:
* UserData
* AuthenticationData

or maybe just `User`, depending on whether you decide to keep the Firebase object exposed